pinot-segment-local/src/main/java/org/apache/pinot/segment/local/io/writer/impl/VarByteChunkForwardIndexWriterV4.java

```diff
@@ -105,11 +107,16 @@ public VarByteChunkForwardIndexWriterV4(File file, ChunkCompressionType compress
     writeHeader(_chunkCompressor.compressionType(), chunkSize);
   }
 
+  // Child class must override this class instance method
+  protected int getConcreteClassVersion() {
```

Comment:
Can we change it back to
```suggestion
  public int getVersion() {
```